DualityEditorPlugins/EditorBase/EditorBasePlugin.cs

```diff
@@ -88,7 +88,7 @@ private void OnResourceModified(ContentRef<Resource> resRef)
 				if (resRef.IsLoaded)
 				{
 					Font fnt = resRef.As<Font>().Res;
-					if (fnt.GlyphsDirty && fnt.EmbeddedTrueTypeFont != null)
+					if (fnt.GlyphsDirty)

```

Comment:
This line should stay as it was before. We shouldn't invoke RenderGlyphs when there isn't actually anything to render. Instead, there could be an else case when `fnt.GlyphsDirty && fnt.EmbeddedTrueTypeFont == null` that only updates kerning specifically.